src/dearpygui_parsers.h

```diff
@@ -579,6 +615,19 @@ InsertParser_Block1(std::map<std::string, mvPythonParser>& parsers)
 		parsers.insert({ "save_image", parser });
 	}
 
+	{
+		std::vector<mvPythonDataElement> args;
+		args.reserve(1);
+		args.push_back({ mvPyDataType::String, "decimal_point" });
+
+		mvPythonParserSetup setup;
+		setup.about = "Change the default decimal_point. Users of non-default decimal point (in particular ',') may be affected by word-selection logic (is_word_boundary_from_right/is_word_boundary_from_left) functions. Use only single character strings.";
+		setup.category = { "Utilities" };
+	
+		mvPythonParser parser = FinalizeParser(setup, args);
+		parsers.insert({ "set_decimal_point", parser });
```

Comment:
I think we need, somewhere in the doc (in a separate article, **not here**), an example of how to use it, and an explanation of why in the world such a thing exists - basically explaining Omar's opinion on locale support in ImGui, as discussed in ImGui tickets [2278](https://github.com/ocornut/imgui/issues/2278) and [6719](https://github.com/ocornut/imgui/issues/6719). In particular, this:

> People may have entire data parsers assuming dot as a separator, etc. Consider the general not-very-internationalization-friendly stance of dear imgui I don't imagine this evolving.

We also need to explain that **both** "," and "." type the decimal separator in the new DPG version (try it on an `add_input_text(scientific=True)`. ImGui (or DPG - for DPG users :grin:) replaces them with that decimal point character.

---

Would it be better if DPG was by default initializing decimal point from `localeconv()`? Maybe not - this might make a horrible mix of English UI and local numeric formats. Anyway, the primary scenario for `set_decimal_point` would be to take the separator from the locale, so we definitely need to document the following invocation:

```py
set_decimal_point(locale.localeconv()["decimal_point"])
```

We could even add an argument like `set_decimal_point(from_locale=True)` to make it look less like black magic (but just having that line in the doc would be sufficient).